src/hatch_openzim/files_install.py

```diff
@@ -0,0 +1,199 @@
+import os
+import shutil
+import tempfile
+import zipfile
+from pathlib import Path
+from typing import Any, Dict, List
+from urllib.request import urlopen
+
+import toml  # to be replaced by tomllib once only 3.11 and above is supported
```

Comment:
should we have a conditional import then?

Reply:
Of course